app/javascript/components/SearchResultsLayout.tsx

```diff
@@ -14,21 +14,26 @@ interface SearchResultsLayoutProps {
 
 const SearchResultsLayout = (props: SearchResultsLayoutProps) => {
   return (
-    <React.Fragment>
+    <>
       <Header 
-        title="Search.gov" 
-      />
-      <Facets />
-      <SearchBar 
-        results={props.results} 
-      />
-      <Results 
-        results={props.results} 
-        vertical={props.vertical}
+        title="Search.gov"
+        isBasic={true} 
```

Comment:
Should we make this a prop instead of using `true`?

Reply:
@ddzz So title I can get from the affiliate. Currently, we dont have a way to know if the header is basic or extended, so I will put a function for this which can be updated once data is available. Will this be fine?
```
title={props.affiliate.display_name}
isBasic={isBasic(props.affiliate)} 
```

Reply:
Actually, since we don't know which header will be used right now, I think it's fine to keep it as `true` for now.